installation/linux/raspbian-raspberry-pi.md

```diff
@@ -5,14 +5,20 @@ Fluent Bit is distributed as **fluent-bit** package and is available for the Ras
 * Raspbian Bullseye \(11\)
 * Raspbian Buster \(10\)
 
-## Server GPG key
+## Server GPG key (old style)
```

Comment:
I'd say this is confusing, let's just do it one way - the old docs are always around if people want the old way.

Reply:
Yup. I understand. Personally, I wanted to add common section to handle apt-key deprecation.